test/TestProject/UnitTest/Fixtures/InventoryFixture.cs

```diff
@@ -14,15 +14,28 @@ public static List<Inventory> BuildInventory(int times)
         {
 
             var invItem = new InventoryItem { ID = 1, product = new Product { ID = 1, Name = "Product", Status = 'A' }, ItemCount = 0, Price = 0, Status = 'A' };
-            inventories.Add(new Inventory {ID= i, items= new List<InventoryItem>{ invItem }, CompletionDate = new DateOnly(2022, 5, 1), ToTalUnits = 1, TotalInventoryPrice = 500.00, Status = 'A'});
+            inventories.Add(new Inventory { ID = i, items = new List<InventoryItem> { invItem }, CompletionDate = new DateOnly(2022, 5, 1), ToTalUnits = 1, TotalInventoryPrice = 500.00, Status = 'A' });
         }
 
         return inventories;
     }
 
+
     public static Inventory BuildInventory()
     {
-        var invItem = new InventoryItem { ID = 1, product = new Product { ID = 1, Name = "Product", Status = 'A' }, ItemCount = 0, Price = 0, Status = 'A' };
-        return new Inventory {ID= 1, items= new List<InventoryItem>{ invItem }, CompletionDate = new DateOnly(2022, 5, 1), ToTalUnits = 1, TotalInventoryPrice = 500.00, Status = 'A'};
+        return BuildInventory(null);
+    }
+
+    public static Inventory BuildInventory(Inventory? inventory)
+    {
+        return new Inventory
+        {
+            ID = inventory?.ID ?? 1,
```

Comment:
@reynoldmorel , I saw that approach before but don't you think is to much work just to avoid the `?` null checking sign? It wouldn't be easier just to handle this situation with the `?`.